include/ideep/operators/conv.hpp

```diff
@@ -126,8 +126,10 @@ struct conv_deconv_utils {
       const auto& src_zero_point = src.has_zero_point() ? src.get_zero_point() :
                                    src_zero_points.empty() ? default_zero_point : src_zero_points;
       const auto& weights_zero_point = weight_grouped.has_zero_point() ? weight_grouped.get_zero_point() : default_zero_point;
-      const auto& dst_zero_point = dst.has_zero_point() ? dst.get_zero_point() :
-                                   dst_zero_points.empty() ? default_zero_point : dst_zero_points;
+      // Similar logic as dst_scales_in. Since when fused with sum, the dst will be the tensor of sum,
+      // In this case, the output tensor' dst_zero_points and dst_scales_in should be passed in explicitly.
+      const auto& dst_zero_point = !dst_zero_points.empty() ? dst_zero_points :
+                                   dst.has_zero_point() ? dst.get_zero_point() : default_zero_point;
```

Comment:
The original logic checks `dst.has_zero_point()` first before `dst_zero_points` which is similar to how the zero point is obtained for `src`. May I know why you are checking `dst_zero_points` before `dst.has_zero_point()` now?

Reply:
It follows the similar logic as `dst_scales_in` and `dst.has_scale()` .https://github.com/intel/ideep/blob/e533c771a1e75a1c225c14b2261eefa62681d9e6/include/ideep/operators/conv.hpp#L142-L143

When fused with sum, the `dst` tensor is actually the `sum` tensor, and `dst` tensor's `get_zero_point()` and `get_scale()` returns `sum` tensor's `zero point` and `scale`. The actual `zero point` and `scale` for `dst` tensor passed through `dst_zero_points` and `dst_scales_in`.

Reply:
But I don't see a check whether it is with sum fusion or not. Should we check whether it is with sum fusion and do things differently?